src/main/java/nus/climods/model/module/Module.java

```diff
@@ -0,0 +1,47 @@
+package nus.climods.model.module;
+
+import java.util.List;
+
+import org.openapitools.client.model.ModuleInformationSemesterDataInner;
+import org.openapitools.client.model.Workload;
+
+/**
+ * A wrapper class for <code>ModuleCondensed</code>
+ */
+public class Module {
+
+    private final org.openapitools.client.model.ModuleInformation apiModuleInfo;
+
+    public Module(org.openapitools.client.model.ModuleInformation apiModuleInfo) {
+        this.apiModuleInfo = apiModuleInfo;
+    }
+
+    public String getTitle() {
+        return apiModuleInfo.getTitle();
+    }
+
+    public String getCode() {
+        return apiModuleInfo.getModuleCode();
+    }
+
+    public String getDepartment() {
+        return apiModuleInfo.getDepartment();
+    }
+
+    public Workload getWorkload() {
+        return apiModuleInfo.getWorkload();
+    }
+
+    public List<ModuleInformationSemesterDataInner> getSemesterData() {
+        return apiModuleInfo.getSemesterData();
+    }
+
+    public int getModuleCredit() {
+        String moduleCreditStr = apiModuleInfo.getModuleCredit();
+        if (moduleCreditStr.isEmpty()) {
+            return 0;
+        }
+
+        return Integer.parseInt(apiModuleInfo.getModuleCredit());
```

Comment:
There is a bug here for modules that have a double/float for module credits e.g YSC2251 with 2.5MCs.
Should we make this return a double instead? i.e `Double.parseDouble(apiModuleInfo.getModuleCredit());`

Reply:
TIL didn't know there's a 2.5 MCs thanks for pointing it out!